pkg/policy/repository.go

```diff
@@ -369,6 +369,8 @@ func (p *Repository) GetJSON() string {
 
 // GetRulesMatching returns whether any of the rules in a repository contain a
 // rule with labels matching the labels in the provided LabelArray.
+// If includeEntities is true, we check if repository contains rules matching
```

Comment:
The commit message seems wrong

Reply:
Changed the wording a bit to make it more clear.